contrib/adapters/ffmpeg_burnins.py

```diff
@@ -313,6 +323,9 @@ def _add_burnin(self, text, align, options, draw):
         }
         data.update(options)
         data.update(_drawtext(align, resolution, text, options))
+        if 'font' in data and _is_windows():
+            data['font'] = data['font'].replace(os.sep, os.sep*2)
```

Comment:
`os.sep*2` is misleading, it should be `r'\'+os.sep` to make it clear that you're escaping the separator (which happens to be a backslash).
Also, is there any harm in escaping these on all platforms?

Reply:
This is for `fontfile=`. When I moved my proxy generation tools back to Windows the single `\` was resulting in `fontfile=C:WindowsFontsarial.ttf` being passed to the subprocess, which is why I needed escapes. I am not sure how the escaping would be useful on *nix systems.